components/suggest/src/suggest.udl

```diff
@@ -95,6 +95,7 @@ dictionary SuggestionQuery {
 
 dictionary SuggestIngestionConstraints {
     u64? max_suggestions = null;
+    sequence<SuggestionProvider>? providers = null;
```

Comment:
Really like the API design here.  It's nice that you can just extend the query options by adding a field.